hack/manifests/base/deployment.yaml

```diff
@@ -60,5 +60,5 @@ spec:
           httpGet:
             path: /actuator/health
             port: 8080
-          initialDelaySeconds: 25
+          initialDelaySeconds: 40
```

Comment:
is this related to the PR? meaning, does not having a specified namespace require more time or this is needed for other issue?